src/sentry/seer/fetch_issues_given_patches.py

```diff
@@ -213,7 +213,14 @@ def _get_issues_for_file(
         tenant_ids={"organization_id": projects[0].organization_id},
         query=query,
     )
-    return raw_snql_query(request, referrer=Referrer.SEER_RPC.value)["data"]
+    try:
+        return raw_snql_query(request, referrer=Referrer.SEER_RPC.value)["data"]
+    except Exception:
+        logger.exception(
```

Comment:
do we have something which can route these to the seer project in sentry instead of the sentry project?